README.md

```diff
@@ -1,24 +1,38 @@
-
 # Validations In Controller Actions Lab
 
-## Objectives
+# Objectives
+
+- Define validations on a model
```

Comment:
i'd like to see more exposition up front to give the user a compelling narrative for the lab. frame it as some sort of actual thing they're building rather than just a spec list